tests/data/input/tim/bc_file_is_incorrect.bc

```diff
@@ -0,0 +1,23 @@
+[forcing]
```

Comment:
What is incorrect about this file?

Reply:
the file itself is not incorrect but it is incorrect for tim format, thus better rename bc_file_format_incorrect_for_timmodel?